stock_measuring_device/wizard/measuring_wizard_line.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+from odoo import _, api, fields, models
```

Comment:
```suggestion
from odoo import api, fields, models
```

same -> `self.env._`